torch/quantization/fake_quantize.py

```diff
@@ -64,23 +64,21 @@ def __init__(self, observer=MovingAverageMinMaxObserver, quant_min=0, quant_max=
 
     @torch.jit.export
     def enable_fake_quant(self, enabled=True):
```

Comment:
these can actually just be global functions instead of residing in the module. maybe create an issue to track this?

Reply:
Filed https://github.com/pytorch/pytorch/issues/44751